frontend/src/safety.ts

```diff
@@ -550,11 +591,19 @@ function specificFileHandling(
       )
       if (partialMatch.length > 0 && fileSystem.partialMatchKey) {
         partialMatch.forEach((x) => {
+          const translationResult = readWriteTranslationKey(x)
+
+          highestDataContainmentLevel = Math.max(
```

Comment:
Might be nicer, to just do this at the end before line 617 based on the contents of `appSafetyRating`?

Reply:
Thanks for the suggestion, implemented!